game/fs.cc

```diff
@@ -12,6 +12,7 @@
 #include <mutex>
 #include <set>
 #include <sstream>
+#include <regex>
```

Comment:
is that realy needed in here?

Reply:
Yup. Within `fs_paths.win.inc` we use a regex and we can't add the include in that file

Reply:
Yup, fs_paths.win.inc has a regex match and the include cannot be in there

Reply:
ok